### PR TITLE
Changed api/v1/files/<storage>/<path> endpoint

### PR DESCRIPTION
**Pull request by Majkluss**

### CODE REVIEW

**Comment by Majkluss** [comment]
SDK first, please @ondratu

File: ChangeLog [hunk 1]
```diff
@@ -27,6 +27,7 @@
     * Fix stats missing for prints of gcodes without M73
     * Fix pause being able to double print time reported
     * Fixed error when trying to get space info of SD Card
+    * Added v1 endpoints for flat filesystem structure, old struct is moved to files_legacy file
```
Comment (ondratu) [inline ChangeLog:30]: Please, add new 0.7.0rc3 header

File: prusa/link/web/files_legacy.py [hunk 2]
```diff
@@ -0,0 +1,670 @@
...
+from ..printer_adapter.prusa_link import TransferCallbackState
+from .lib.auth import check_api_digest
+from .lib.core import app
+from .lib.files import (file_to_api, gcode_analysis, get_os_path, local_refs,
+                        sdcard_refs, sort_files)
+
+log = logging.getLogger(__name__)
+
+
+def check_filename(filename):
```
Comment (ondratu) [inline prusa/link/web/files_legacy.py:41]: Are you sure, that this functions will be used only in this legacy file? If no, please add (leave) them in web/files.py file.

**Review by ondratu — CHANGES_REQUESTED** [review]

File: prusa/link/web/files_legacy.py (hunk 2 above)
Comment (Majkluss) [reply prusa/link/web/files_legacy.py:41]: I didn't delete them

**Review by ondratu — APPROVED** [review]

**Review by ondratu — APPROVED** [review]

**Pull request merged**